Source/JavaScriptCore/heap/Heap.cpp

```diff
@@ -1674,6 +1670,7 @@ NEVER_INLINE bool Heap::runEndPhase(GCConductor conn)
         snapshotUnswept();
         finalizeUnconditionalFinalizers(); // We rely on these unconditional finalizers running before clearCurrentlyExecuting since CodeBlock's finalizer relies on querying currently executing.
         removeDeadCompilerWorklistEntries();
+        deleteUnmarkedCompiledCode();
```

Comment:
We move this to here which is calling `finalizeUnconditionalFinalizers` etc., since this is the place we can use `isMarked` to check the liveness.

Reply:
> I also see that previously, we only call this while inside a SweepingScope. But since we're now calling it in the EndPhase where the world is stopped, then I presume we don't need to worry about that. Can you confirm?

Yes, that's the main reason of moving this here.

Reply:
Moved.